packages/core-ethereum/src/indexer/index.spec.ts

```diff
@@ -371,6 +373,9 @@ describe('test indexer', function () {
     newBlock()
     newBlock()
     await opened.promise
+    // Total hack as this test sucks. There is no way to await the actual
+    // commitment setting as this is behind an event.
+    await new Promise((resolve) => setTimeout(resolve, 2000))
```

Comment:
This should work by `await`ing `commitmentSet.promise`. Timeouts shouldn't be necessary. Didn't it work before?

Reply:
It _should_. It doesnt. The comment explains.